### DISCUSSION TITLE
Community Profile Items Not Recognized

### DISCUSSION THREAD

**Discussion by jsjoeio** [community/community#24011] [New to GitHub]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/5/8/58499d67a5d8f34019bab841d5cb4a3659a4ea56.jpeg" data-download-href="/uploads/short-url/cB1JFaiKFZvVKFd9LbcHtYrUufc.jpeg?dl=1" title="Screen Shot 2021-02-08 at 2.42.15 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/3806031/181080630-689ca1c7-c33e-42f1-90ef-47d969b33179.jpeg" alt="Screen Shot 2021-02-08 at 2.42.15 PM" data-base62-sha1="cB1JFaiKFZvVKFd9LbcHtYrUufc" width="690" height="396" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/8/58499d67a5d8f34019bab841d5cb4a3659a4ea56_2_690x396.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/5/8/58499d67a5d8f34019bab841d5cb4a3659a4ea56_2_1035x594.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/5/8/58499d67a5d8f34019bab841d5cb4a3659a4ea56_2_1380x792.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/8/58499d67a5d8f34019bab841d5cb4a3659a4ea56_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2021-02-08 at 2.42.15 PM</span><span class="informations">4064×2334 483 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>For some reason, GitHub is not recognizing a few items in the Community profile for our repository code-server. We have the following:</p>
<ul>
<li><a href="https://github.com/cdr/code-server/blob/41ad0c0c4c14df78e1bfa3a44d8b633ed2fc26e5/README.md" rel="noopener nofollow ugc">README</a></li>
<li><a href="https://github.com/cdr/code-server/blob/41ad0c0c4c14df78e1bfa3a44d8b633ed2fc26e5/docs/CODE_OF_CONDUCT.md" rel="noopener nofollow ugc">Code of conduct</a></li>
<li><a href="https://github.com/cdr/code-server/blob/41ad0c0c4c14df78e1bfa3a44d8b633ed2fc26e5/docs/CONTRIBUTING.md" rel="noopener nofollow ugc">Contributing</a></li>
</ul>
<p>Any ideas on how to get GitHub to look in the correct place?</p>

**Comment by jsjoeio** [comment]
<p>Based on the wording <a href="https://docs.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file#about-default-community-health-files" rel="noopener nofollow ugc">here</a>:</p>
<blockquote>
<p>GitHub will use and display default files for any public repository owned by the account that does not have its own file of that type in any of the following places:</p>
<p>the root of the repository<br>
the .github folder<br>
the docs folder</p>
</blockquote>
<p>It sounds like it can be under <code>/docs</code> but not sure.</p>

**Comment by canuckjacq** [comment]
<p>:wave:</p>
<p>Thanks for letting us know. I’ve had a look and we have some other reports of the same behaviour. We’ve opened an issue with the team and I’ll let you know when there is news.</p>

**Comment by BenHenning** [comment]
<p>Another data point if it helps: <a href="https://github.com/oppia/oppia-android" class="inline-onebox" rel="noopener nofollow ugc">GitHub - oppia/oppia-android: A free, online &amp; offline learning platform to make quality education accessible for all.</a> &amp; <a href="https://github.com/oppia/oppia-android/community" class="inline-onebox" rel="noopener nofollow ugc">Community · GitHub</a>. We have all of the requested documentation under our .github folder (though our code of conduct is slightly modified, so per <a href="https://github.community/t/code-of-conduct-is-present-in-repo-but-insights-community-tab-doesnt-detect-it/1534/7" class="inline-onebox">Code of Conduct is present in repo - but Insights -&gt; Community tab doesn't detect it? - #7 by jab</a> I expect that one will fail).</p>

**Comment by yamiacat** [comment, marked as answer]
<p><a class="mention" href="/jsjoeio">@jsjoeio</a> - We’ve now patched the bug in the community panel that was causing intermittent detection of some of the files, so you should now be seeing green checks as expected!</p>
<p><a class="mention" href="/benhenning">@BenHenning</a> - we’ve also extended what gets recognized as a Code of Conduct, so you should be seeing a green check mark there, too!</p>

**Comment by jsjoeio** [comment]
<p>I can confirm that it is indeed working! Thanks <a class="mention" href="/yamiacat">@yamiacat</a> :smile:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/1/114d83ffc7437e19c403324dcfcaee80ac969a31.png" data-download-href="/uploads/short-url/2t4bZCNbS6KBnEBSYSmmmdPSen7.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/3806031/181080632-288447cf-fde2-4690-a55d-ddda2570c48c.png" alt="image" data-base62-sha1="2t4bZCNbS6KBnEBSYSmmmdPSen7" width="402" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/1/114d83ffc7437e19c403324dcfcaee80ac969a31_2_402x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/1/114d83ffc7437e19c403324dcfcaee80ac969a31_2_603x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/1/114d83ffc7437e19c403324dcfcaee80ac969a31_2_804x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/1/114d83ffc7437e19c403324dcfcaee80ac969a31_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">840×1044 30.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
